### ISSUE TITLE
Add support for DRBD 8.4

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#268]
Originally reported of Google Code with ID 212.

```
What software version are you running? Please provide the output of "gnt-
cluster --version" and "gnt-cluster version".

[root@ne1 ganeti]# gnt-cluster --version
gnt-cluster (ganeti v2.4.5) 2.4.5

[root@ne1 ganeti]# gnt-cluster version
Software version: 2.4.5
Internode protocol: 2040000
Configuration format: 2040000
OS api version: 20
Export interface: 0


<b>What distribution are you using?</b>

CentOS 6.2

<b>What steps will reproduce the problem?</b>

[root@ne1 ganeti]# gnt-instance add -n ne1.domain.net:ne2.domain.net -o debootstrap -t drbd -s 100G ganeti_base.domain.net
Mon Jan 30 08:53:48 2012 * creating instance disks...
Mon Jan 30 08:53:49 2012  - WARNING: Device creation failed, reverting...
Failure: command execution error:
Can't create block device <DRBD8(hosts=ne1.domain.net/0-ne2.domain.net/0, port=11025, configured as 10.1.1.195:11025 10.1.1.196:11025, backend=<LogicalVolume(/dev/vmvg/384ed310-d2ec-4fd7-8991-a1581d6f8c9b.disk0_data, not visible, size=102400m)>, metadev=<LogicalVolume(/dev/vmvg/384ed310-d2ec-4fd7-8991-a1581d6f8c9b.disk0_meta, not visible, size=128m)>, visible as /dev/disk/0, size=102400m)> on node ne1.domain.net for instance ganeti_base.domain.net: Can't assemble device after creation, unusual event: drbd0: can't attach local disk: 
USAGE: drbdsetup command device arguments options

Device is usually /dev/drbdX or /dev/drbd/X.


<b>What is the expected output? What do you see instead?</b>

Expecting the instance to be created. Not sure what the exact output is.
This is most likely my error, but I need some help figuring out what I did wrong.


Some extra information:

[root@ne1 ganeti]# vgdisplay
  --- Volume group ---
  VG Name               vmvg
  System ID             
  Format                lvm2
  Metadata Areas        1
  Metadata Sequence No  83
  VG Access             read/write
  VG Status             resizable
  MAX LV                0
  Cur LV                0
  Open LV               0
  Max PV                0
  Cur PV                1
  Act PV                1
  VG Size               372.52 GiB
  PE Size               4.00 MiB
  Total PE              95365
  Alloc PE / Size       0 / 0   
  Free  PE / Size       95365 / 372.52 GiB
  VG UUID               x9ug3U-cNEt-SPs7-hzO8-ZpNe-YrQH-aQ39po

DRBD version is 8.4.
```
Originally added on 2012-01-30 14:16:08 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
Unfortunately we don't support yet DRBD 8.4 - the syntax has changed significantly and we haven't adapted Ganeti yet for it.
```
Originally added on 2012-01-30 15:24:51 +0000 UTC.

Added Labels: Type-Enhancement Priority-Medium OpSys-All
Changed State: Accepted

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2012-07-19 14:53:23 +0000 UTC.

Added Labels: Component-storage

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2012-10-01 10:49:15 +0000 UTC.

Now blocking: 273 (now #329);

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2013-02-13 14:42:39 +0000 UTC.

Added to Milestone: Release2.9

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2013-04-24 06:44:03 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2013-05-07 14:10:13 +0000 UTC.

Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
In case anybody else is puzzled by the changes without comments, this has been fixed in ganeti 2.9.0 beta1:

Version 2.9.0 beta1
*(Released Thu, 29 Aug 2013)*
- DRBD 8.4 support. Depending on the installed DRBD version, Ganeti now uses
  the correct command syntax. It is possible to use different DRBD versions
  on different nodes as long as they are compatible to each other. This
  enables rolling upgrades of DRBD with no downtime. As permanent operation
  of different DRBD versions within a node group is discouraged,
  ``gnt-cluster verify`` will emit a warning if it detects such a situation.
```
Originally added on 2013-09-20 16:07:33 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
Ganeti 2.9.0 has been released
```
Originally added on 2013-11-05 13:56:36 +0000 UTC.

Changed State: Released
